testing/README.md

```diff
@@ -0,0 +1,219 @@
+# Implementation Report
+
+This directory contains the
+[WoT Thing Description Implementation report](https://w3c.github.io/wot-thing-description/testing/report.html)
+and the sources for it.
+The report itself is in [report.html](report.html).
+Do not edit this file,
+it is autogenerated by executing `npm run assertions` from the main directory,
+and any edits would be overwritten the next time you generate it.
+
+## Adding Your Results
+
+If you want to update your input for a Member organization, please go to [WoT IG testing](https://github.com/w3c/wot/tree/master/testing/tests/2019-05) and see the [README](https://github.com/w3c/wot/blob/master/testing/tests/2019-05/README.md) there.
+
+## Updating Implementation Report
+
+### w3c/wot Repository: Updating Results from Inputs
+
+Test inputs are collected via the WoT IG and are currently in `wot/testing/tests/2019-05`.
+
+---
+
+#### First Time Only
+To update a test run to incorporate new results or test cases, first check out the `wot` repo,
+```
+git clone https://github.com/w3c/wot.git
+```
+and install the dependencies for the AssertionTester tool:
+```
+cd wot/testing/tools/thingweb-playground/AssertionTester
+npm install
+```
+
+---
+
+Go to the `wot/testing/tests/2019-05` directory.
```

Comment:
2019?

Reply:
old file, this whole section is gone now (we are going to describe the data collection process in wot-testing, not in this readme)